API.md

```diff
@@ -0,0 +1,19 @@
+# Pyflow commands API
```

Comment:
While refactoring is not merged - this is the file that interests us for this PR.